Thai/urbex_bkk

```diff
@@ -0,0 +1,10 @@
+Have you ever explored a deserted building in Bangkok ?
```

Comment:
Hi,

   Thank you for notifying me. I was oblivious to do so. I will make changes soon.

Sent from my iPhone

> On 11 ส.ค. 2558, at 02:52, Casey Collaborator <notifications@github.com> wrote:
> 
> In Thai/urbex_bkk:
> 
> > @@ -0,0 +1,10 @@
> > +Have you ever explored a deserted building in Bangkok ?
> It looks like this page is not reachable from the rest of the story. That is, if a reader starts in language.md and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.
> 
> —
> Reply to this email directly or view it on GitHub.
>